src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -76,9 +76,15 @@ def legacy_dir(self):
 
     @property
     def legacy_config_path(self):
-        m = md5()
-        m.update(self.base_path)
-        meta_config = "{0}.meta".format(m.digest().hex())
+        try:
+            m = hashlib.md5(self.base_path)
+        except ValueError:
+            try:
+                m = hashlib.md5(self.base_path, usedforsecurity=False) # type: ignore
+            except TypeError:
+                raise VolumeException(-errno.EINVAL, "usedforsecurity flag is not supported on this system and FIPs enabled.")
```

Comment:
Maybe saying, "require python's hashlib library to support usedforsecurity flag in FIPS enabled systems", is more useful?